src/gatk_gcnv_caller.sh

```
@@ -0,0 +1,655 @@
+if (( IO_PROCESSES > 32 )); then IO_PROCESSES=32; fi
+
+
+main() {
+    mark-section "Installing packages and loading GATK Docker image"
+    sudo dpkg -i sysstat*.deb
+    sudo dpkg -i parallel*.deb
+    sudo -H python3 -m pip install --no-index --no-deps packages/*
+
+    dx download "$GATK_docker" -o GATK.tar.gz
+    docker load -i GATK.tar.gz
+    export GATK_image=$(docker images --format="{{.Repository}} {{.ID}}" | grep "^broad" | cut -d' ' -f2)
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)